create_framework/dependency_injection.rst

```diff
@@ -227,16 +227,16 @@ object::
     $container->setParameter('charset', 'UTF-8');
 
 Instead of relying on the convention that the routes are defined by the
-``$routes`` variables, let's use a parameter again::
+``$routes`` variables, let's use a reference::
 
     // ...
     $container->register('matcher', Routing\Matcher\UrlMatcher::class)
-        ->setArguments(['%routes%', new Reference('context')])
+        ->setArguments([new Reference('routes'), new Reference('context')])
     ;
 
 And the related change in the front controller::
 
-    $container->setParameter('routes', include __DIR__.'/../src/app.php');
+    $container->set('routes', $routes);
```

Comment:
Not sure if it's clear enough for readers what `$routes` needs to be. Do we need to clarify it?

Reply:
`$routes` variable already exists and is defined above with `$routes = include __DIR__.'/../src/app.php';`